vale/code/arch/ppc64le/Vale.PPC64LE.InsBasic.vaf

```diff
@@ -124,7 +124,17 @@ procedure LoadImm64(out dst:reg_opr, inline src:simm16)
     {:quick exportOnly}
     {:instruction Ins(S.LoadImm64(dst, src))}
     ensures
-        dst == old(int_to_nat64(src));
+        dst == old(src % pow2_64);
```

Comment:
This is a minor comment, but if you can use addition and subtraction in place of mod-large-number for the `.vaf` instruction definitions, it will sometimes speed up verification.  This is why `Add64Wrap` uses `add_wrap`, defined as `if x + y < n then x + y else x + y - n`, rather than `(x + y) % pow2_64`.

Reply:
I was missing the usage of `sub_wrap` for subtraction instructions so I replaced the old procedure in the new commit. Regarding mod-large-number of inline integers, I can replace them with `nat` but we will omit the boundaries that have the same behavior of checking instruction parameters by assembler at compile-time https://github.com/hacl-star/hacl-star/blob/36ab944703a52b52218bce295614d15fc09b22f3/vale/code/arch/ppc64le/Vale.PPC64LE.InsBasic.vaf#L109-L110